internal/api_client/sddc_manager_client.go

```diff
@@ -234,17 +214,44 @@ func (sddcManagerClient *SddcManagerClient) getTask(ctx context.Context, taskId
 	}
 	// reset the counter
 	sddcManagerClient.getTaskRetries = 0
-	return getTaskResult.Payload, nil
+
+	return res.JSON200, nil
```

Comment:
JSON200?

Reply:
status code handling has been generalized in a utility function